tests/test_aggregator.py

```diff
@@ -0,0 +1,73 @@
+import logging
```

Comment:
Could you please leave a TODO here, that this needs to be use in an integrationtest? Thanks!

Reply:
Sure, leaving it now